src/style.css

```
@@ -1 +1,73 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  --max__height: 60px;
+  --active_color_link: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  max-height: var(--max__height);
+  box-sizing: border-box;
+}
+
+.logo {
+  display: block;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  line-height: 14px;
+  display: flex;
+}
+
+.nav__element_list-item {
+  list-style: none;
+  margin: 0 10px;
+}
+
+.nav__element_list-item:last-child {
+  margin-right: 0;
+}
+
+.nav__element_list-item:first-child {
+  margin-left: 0;
+}
```

you can just do like this
```suggestion
.nav__item {
  list-style: none;
}

.nav__item:not(:last-child) {
  margin-right: 20px;
}
```

---

Wow, this is a very interesting idea, i had a lot of troubles with tests before i used separate margins, thanks a lot!